File: chc-ccr-csw-master/chc-ccr-clientsw-dashboard/chc-ccr-clientsw-dashboard-ui/src/app/modules/configuration/addon/dto/addon-config-dto.ts
import { CommonConfigDTO, CommonConfigFilterDTO } from "../../common-config-dto";

export class AddonConfigFilterDTO extends CommonConfigFilterDTO{
   
}

export class AddonConfigDTO extends CommonConfigDTO{
    sndPath!: string;
    rcvPath!: string;
    sendingPrefix!: string;
    errorPrefix!: string;
    sentPrefix!: string;
    errorDeliverPrefix!: string;
    csc!: string;
}

File: chc-ccr-csw-master/chc-ccr-clientsw-dashboard/chc-ccr-clientsw-dashboard-api/src/main/java/com/cbi/ccr/csw/dashboard/addon/dto/AddonConfigDTO.java
package com.cbi.ccr.csw.dashboard.addon.dto;

import com.cbi.frw.api.dto.GenericDTO;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class AddonConfigDTO extends GenericDTO{
	private Long id;
	private String localBaId;
	private String remoteBaId;
	private String sndPath;
	private String rcvPath;
	private String sendingPrefix;
	private String errorPrefix;
	private String sentPrefix;
	private String errorDeliverPrefix;
}
